invenio_accounts/models.py

```diff
@@ -463,3 +468,153 @@ def delete_by_user(cls, method, user):
         """Unlink a user from an external id."""
         with db.session.begin_nested():
             cls.query.filter_by(id_user=user.id, method=method).delete()
+
+
+class DomainOrg(db.Model):
```

Comment:
Needed for being able to do statistics on different domains. E.g. a typical case is email domains like ``physics.uni.org`` and ``bio.uni.org`` all linking to the same organisation.